libs/sdk-backend-bear/src/utils/api.ts

```diff
@@ -208,17 +208,25 @@ export const updateUserMap = async (
 
     const results = await Promise.all(
         uniqueUsersToLoad.map((uri) => {
-            return authCall(async (sdk): Promise<IUser> => {
-                const result = await sdk.xhr.getParsed<GdcUser.IWrappedAccountSetting>(uri);
-                return convertUser(result.accountSetting);
+            return authCall(async (sdk): Promise<IUser | undefined> => {
+                try {
+                    const result = await sdk.xhr.getParsed<GdcUser.IWrappedAccountSetting>(uri);
+                    return convertUser(result.accountSetting);
+                } catch (ex) {
+                    // Inactive user return Forbidden error
```

Comment:
This will be confusing in a few months :) what about something like
```ts
// for inactive users, non-admins will get Forbidden from the server
// so on error we assume that the user that was requested is no longer there (is inactive)
```

Reply:
Also, should we check for the Forbidden? What if the server returns 500 or something else?

Reply:
than will explode whole dashboard :-) No this is hot fix (QA has down all scenarios) We have to refactor this.

Reply:
fixed